components/rendering/src/omeis/providers/re/quantum/Quantization_32_bit.java

```diff
@@ -213,7 +212,6 @@ private int _quantize(double value)
     public Quantization_32_bit(QuantumDef qd, Pixels pixels) {
         super(qd, pixels);
         values = CacheBuilder.newBuilder()
-                .maximumSize(MAX-MIN+1)
```

Comment:
Not for this PR, but we might want to move all these specifications out to etc/omero.properties so that if they start to affect memory, a sysadmin could lower them. cc: @mtbc cf:
 * https://github.com/joshmoore/openmicroscopy/blob/10661-dynamic-params/components/blitz/src/ome/services/blitz/util/ParamsCache.java#L105
 * https://github.com/joshmoore/openmicroscopy/blob/10661-dynamic-params/etc/omero.properties#L196